internal/installer/elasticagent_tar.go

```diff
@@ -46,7 +49,7 @@ func (i *elasticAgentTARPackage) AddFiles(ctx context.Context, files []string) e
 func (i *elasticAgentTARPackage) Inspect() (deploy.ServiceOperatorManifest, error) {
 	return deploy.ServiceOperatorManifest{
 		WorkDir:    "/opt/Elastic/Agent",
-		CommitFile: "/elastic-agent/.elastic-agent.active.commit",
+		CommitFile: "elastic-agent/.elastic-agent.active.commit",
```

Comment:
I do not know how many times we have done this change, and reverted it 😄 Hopefully the team will be consistent in the future